lmms_eval/tasks/videomme/videomme_w_subtitle.yaml

```diff
@@ -25,7 +25,7 @@ metric_list:
     higher_is_better: true
 lmms_eval_specific_kwargs:
   default:
-    frame_num: 32
+    frame_num: 768
```

Comment:
You can keep the default one unchanged and set frame num specific for `llava_vid`